scripts/cc-cli/src/commands/unbond.ts

```diff
@@ -22,14 +23,22 @@ async function unbondAction(options: OptionValues) {
   // Check options
   checkAmount(options);
 
-  // Build account
   const seed = getSeedFromOptions(options);
-  const stash = initKeyringPair(seed);
+  const controllerKeyring = initKeyringPair(seed);
+  const controllerAddress = controllerKeyring.address;
+
+  const controllerStatus = await getStatus(controllerAddress, api);
+  if (!controllerStatus.stash) {
+    console.error(`Cannot unbond, ${controllerAddress} is not staked`);
+    process.exit(1);
+  }
```

Comment:
Not sure what's happening here:

- Using correct controller seed for my V2 account
- V2 has been slashed and isn't actively validating

```
$ node dist/index.js show-address -u ws://127.0.0.1:9946 -s "twice wash veteran endorse water orbit fit casino flame provide cancel peace"
Account address: 5EEoJZYPpTYGX1t3shaQbqhRmqvdvoaTycY1gJ4TB8spDgPp

$ node dist/index.js status -u ws://127.0.0.1:9946 -a 5EEoJZYPpTYGX1t3shaQbqhRmqvdvoaTycY1gJ4TB8spDgPp
2023-06-22 20:13:05        API/INIT: RPC methods not decorated: creditcoin_hashrate, task_getOffchainNonceKey
2023-06-22 20:13:05        API/INIT: creditcoin-node/222: Not decorating unknown runtime apis: TaskApi/1
Bonded:  false
Stash:  5FEK97W73jQHWdUYAeZvyb7iRuygZhFfC75ZA9cfqkQ8yhvb
Controller:  None
Validating:  false
Waiting:  false
Active:  false
Next unbonding chunk: None

$ node dist/index.js unbond -u ws://127.0.0.1:9946 -s "twice wash veteran endorse water orbit fit casino flame provide cancel peace" -a 1000
2023-06-22 20:13:26        API/INIT: RPC methods not decorated: creditcoin_hashrate, task_getOffchainNonceKey
2023-06-22 20:13:26        API/INIT: creditcoin-node/222: Not decorating unknown runtime apis: TaskApi/1
Unbond transaction hash: 0xd6b1800c0b0de740765a82ef10be934398eb19e56d91a7b0d95cede8689ec2ed
```

I was expecting not to be able to unbond. 

Meanwhile Polkadot JS APPS shows
![image](https://github.com/gluwa/creditcoin/assets/1002300/31fe563b-a377-45bb-90f0-2956a727d501)

After posting the screenshot:

```
$ node dist/index.js status -u ws://127.0.0.1:9946 -a 5EEoJZYPpTYGX1t3shaQbqhRmqvdvoaTycY1gJ4TB8spDgPp
2023-06-22 20:17:53        API/INIT: RPC methods not decorated: creditcoin_hashrate, task_getOffchainNonceKey
2023-06-22 20:17:53        API/INIT: creditcoin-node/222: Not decorating unknown runtime apis: TaskApi/1
Bonded:  false
Stash:  5FEK97W73jQHWdUYAeZvyb7iRuygZhFfC75ZA9cfqkQ8yhvb
Controller:  None
Validating:  false
Waiting:  false
Active:  false
Next unbonding chunk: None


$ node dist/index.js unbond -u ws://127.0.0.1:9946 -s "twice wash veteran endorse water orbit fit casino flame provide cancel peace" -a 2000
2023-06-22 20:17:59        API/INIT: RPC methods not decorated: creditcoin_hashrate, task_getOffchainNonceKey
2023-06-22 20:17:59        API/INIT: creditcoin-node/222: Not decorating unknown runtime apis: TaskApi/1
Unbond transaction hash: 0x11ff6a16b9bdd45452e6caabb68b268473a477f4b4868ad8cdf6a4040e172397
```

![image](https://github.com/gluwa/creditcoin/assets/1002300/cc5aa69f-dd1e-497c-8b78-d2200f75c451)

Reply:
Ignore ^^^^.

I was using the controller address for status.  Looks good with Stash address:

```
$ node dist/index.js show-address -u ws://127.0.0.1:9946 -s "acid popular pretty action now super crime amount neutral height keen diet"
Account address: 5FEK97W73jQHWdUYAeZvyb7iRuygZhFfC75ZA9cfqkQ8yhvb

$ node dist/index.js status -u ws://127.0.0.1:9946 -a 5FEK97W73jQHWdUYAeZvyb7iRuygZhFfC75ZA9cfqkQ8yhvb
2023-06-22 20:21:38        API/INIT: RPC methods not decorated: creditcoin_hashrate, task_getOffchainNonceKey
2023-06-22 20:21:38        API/INIT: creditcoin-node/222: Not decorating unknown runtime apis: TaskApi/1
Bonded:  true
Stash:  undefined
Controller:  5EEoJZYPpTYGX1t3shaQbqhRmqvdvoaTycY1gJ4TB8spDgPp
Validating:  false
Waiting:  false
Active:  false
Next unbonding chunk: 929 CTC in 7 minutes, 20 seconds
```

Reply:
Using V3 controller which is not bonded:

```
$ node dist/index.js chill -u ws://127.0.0.1:9946 -s "symbol endless bitter mosquito usage trigger soup later maximum girl awake sign" 
Cannot chill, 5Eq3uWk9WAzPt4RRVXg1cuNWZN7StujipUg5V7Kme1YN2P7F is not bonded

$ node dist/index.js unbond -u ws://127.0.0.1:9946 -s "symbol endless bitter mosquito usage trigger soup later maximum girl awake sign" -a 9
Cannot unbond, 5Eq3uWk9WAzPt4RRVXg1cuNWZN7StujipUg5V7Kme1YN2P7F is not staked
```